python/Upload_CV_List.py

```diff
@@ -0,0 +1,42 @@
+# Preliminaries
+import pandas as pd
+import argparse
+import synapseclient
+from synapseclient import Project, File, Folder
+from synapseclient import Schema, Column, Table, build_table, Row, RowSet, as_table_columns
+from pathlib import Path
+
+# Login to  Synapse
+syn = synapseclient.Synapse()
+syn.login()
+
+# Get Arguments - Synapse project id and file path
+parser = argparse.ArgumentParser(description = 'Get synapse project id, file path, and name of table')
```

Comment:
Per PEP8's guidelines, do not include whitespace around the `=`for arguments:

<img width="788" alt="Screen Shot 2021-03-11 at 21 14 56" src="https://user-images.githubusercontent.com/9377970/110895812-34b38500-82af-11eb-8c5d-bcad6ab0b4a9.png">

This line is also longer than 79 characters.

```suggestion
parser = argparse.ArgumentParser(
    description='Get synapse project id, file path, and name of table')
```